.github/workflows/build.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - '*'
```

Comment:
Got a syntax warning for leaving this blank, so just put simple regex - that being said, maybe it would make more sense to have this be whatever the latest release tag is, and make sure it gets updated in the [Dask version updating workflow](https://github.com/dask/dask-docker/blob/main/.github/workflows/watch-conda-forge.yml)?

Reply:
I'm happy with the wildcard here.